presets/4.4/osd/YarosFPV_DJI_O3_OSD.txt

```diff
@@ -13,13 +14,13 @@
 #$ DESCRIPTION:
 #$ DESCRIPTION: Information
 #$ DESCRIPTION: -----------
-#$ DESCRIPTION: 
+#$ DESCRIPTION:
 #$ DESCRIPTION: DJI HD OSD setup without GPS. Set the OSD mode set to HD in your DJI Goggles for Betaflight elements to display. You will also need to select "VTX (MSP + Displayport) on the appropriate port in the ports tab.
-#$ DESCRIPTION: 
+#$ DESCRIPTION:
 #$ DESCRIPTION: Preview:
 #$ DESCRIPTION: -----------
-#$ DESCRIPTION: <img src="https://i.ibb.co/GtD9WMm/Screenshot-2024-06-20-195831.jpg" style="width:100%"/>
-#$ DESCRIPTION: 
+#$ DESCRIPTION: <img src="https://i.ibb.co/GtD9WMm/Screenshot-2024-06-20-195831.jpg" width="400px" style="margin-left: auto; margin-right: auto; display: block;"/>
```

Comment:
Does this scale on different screen sizes?

Reply:
@haslinghuis I did this because the `width:100%` approach didn't render in BF
look here:
<img width="615" alt="Screenshot 2024-08-16 at 16 07 51" src="https://github.com/user-attachments/assets/18060dd0-8846-414d-87dd-c3a972d3c18d">

Reply:
is it sorted if so merge @haslinghuis ?